SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -10,65 +10,111 @@
 
 #include "AssetRegistryModule.h"
 #include "GeneralProjectSettings.h"
+#include "Misc/ScopedSlowTask.h"
+#include "UObject/StrongObjectPtr.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialGDKEditor);
 
-FSpatialGDKEditor::FSpatialGDKEditor()
-	: bSchemaGeneratorRunning(false)
-{
-	TryLoadExistingSchemaDatabase();
-}
+#define LOCTEXT_NAMESPACE "FSpatialGDKEditor"
 
-void FSpatialGDKEditor::GenerateSchema(FSimpleDelegate SuccessCallback, FSimpleDelegate FailureCallback, FSpatialGDKEditorErrorHandler ErrorCallback)
+bool FSpatialGDKEditor::GenerateSchema(bool bFullScan)
 {
 	if (bSchemaGeneratorRunning)
```

Comment:
I don't think this variable is explicitly initialized anywhere, which can sometimes result in failures saying that schema generation is already running. I've just had it happen when running from the commandlet.

Reply:
Done